jdaviz/configs/specviz/helper.py

```diff
@@ -144,7 +144,7 @@ def _set_scale(self, scale, axis, min_val=None, max_val=None):
             elif isinstance(min_val, u.Quantity):
                 # Convert user's value to axis' units
                 min_val = min_val.to(axis.unit).value
-            # If auto, set to min_valimum wavelength value
+            # If auto, set to min axis wavelength value
```

Comment:
I think someone tried to replace "minimum" with "min_val" but gave up half-way...

```suggestion
            # If auto, set to min_val wavelength value
```

Reply:
But your change makes more sense...